app/templates/download.html

```diff
@@ -0,0 +1,27 @@
+{% from 'macros/page_heading.html' import page_heading %}
```

Comment:
@robyoung We could, I assumed there was some reasoning behind downloading being a separate action.  If there isn't I'm happy to change it though.